src/Query/Grammar.php

```diff
@@ -195,4 +195,17 @@ public function supportsSavepoints()
     {
         return false;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function substituteBindingsIntoRawSql($sql, $bindings)
```

Comment:
This is the wrong place to fix this. You should fix it in `Connection::escape()` https://github.com/colopl/laravel-spanner/blob/c21ad52052fc7ba79e088cb73bfcff41a5774f52/src/Connection.php#L450.

Reply:
Sorry, fixed it